packages/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -239,9 +240,38 @@ describe('SendConfirmation', () => {
   it('renders correct modal for invitations', async () => {
     const { getByTestId, queryAllByTestId } = renderScreen({}, mockInviteScreenProps)
 
-    expect(queryAllByTestId('InviteAndSendModal')[0].props.visible).toBe(false)
-    // Fire event press not working here so instead we call the onClick directly
-    getByTestId('ConfirmButton').props.onClick()
```

Comment:
Why removing? it checked the modal wasn't present before clicking, which seems is still relevant.

Reply:
Yup, I don't know what I was thinking, I may have deleted it by mistake